src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -62,24 +62,26 @@ public static bool TryParseBoolEditorConfigCodeStyleOption(string arg, CodeStyle
         public static bool TryGetCodeStyleValueAndOptionalNotification(
```

Comment:
i don't know how it works, but we often use a 'PerformanceSensitive' attribute on code that we've done something like thsi for.  @sharwell can weigh in on if that should be used here and the right way to configure it.